packages/react-refresh/src/ReactFreshBabelPlugin.js

```diff
@@ -140,9 +140,6 @@ export default function(babel, opts = {}) {
             } else if (calleeType === 'MemberExpression') {
               // Could be something like React.forwardRef(...)
               // Pass through.
-            } else {
```

Comment:
from what I understand this was just an optimization as the declared binding is being tested later more to check if it should be registered anyway